welly/well.py

```diff
@@ -325,17 +268,96 @@ def from_las(cls,
                 raise WellError('Could not retrieve url: ', e)
             fname = (StringIO(data))
 
-        las = lasio.read(fname, encoding=encoding)
+        data_sets = from_las(fname, encoding=encoding)
+
+        # unpack datasets
+        for name, (df_data, df_header) in data_sets.items():
```

Comment:
do we really want to keep all this logic in the `well` class?

Reply:
Will move this out in #6 because the construction logic will be influenced by changing the `Curve` object type.